docs/target.settings.yaml

```diff
@@ -4,7 +4,18 @@ general:
   cm_device_codename: grouper
   cm_release_type: snapshot
   cm_release_version: M12
-  parent_template: mia-default
+  parent_templates:
+    - mia-default
+    - wifi-only
```

Comment:
We should one day rename `mia-default` to `cm11-default` (and maybe add the CPU architecture) or something on those lines.

And then we would have a `cm11-wifi-only` template with this:
```
parent_templates:
  - cm11-default
```

But the created definition should have something on the lines
```
parent_templates:
  - cm11-default
  - cm11-wifi-only
  - cm11-alternative-keyboard
```
And then if you use the `mia definition update-from-template <definition>` (added by mike in https://github.com/mikeperry-tor/mission-impossible-android/commit/9691b70c6a957750f16168ed09615e14d7e943e0): it would go through all of them in order.